turbot/__init__.py

```diff
@@ -513,12 +504,10 @@ def buy_command(self, channel, author, params):
 
     def reset_command(self, channel, author, params):
         """
-        DO NOT USE UNLESS ASKED. Generates a final graph for use with !lastweek and
-        resets all data for all users.
+        Only Turbot Admin members can run this command. Generates a final graph for use
+        with !lastweek and resets all data for all users.
         """
-
-        # Only users with the Turbot Admin role can do a reset
-        if not is_turbot_admin(channel, author):
+        if not is_turbot_admin(author):
```

Comment:
Moved your comment into the docstring, seemed like a good place for it to live.

Reply:
Oops, I remember thinking I needed to update that when I was making the changes, but then I got distracted by turnips